src/packages/frontend/editors/task-editor/list.tsx

```diff
@@ -147,7 +148,8 @@ export default function TaskList({
   }
 
   function on_click(e) {
-    if (e.target === mainDivRef.current) {
+    // test, if e.target is a child of mainDivRef.current
+    if (mainDivRef.current.contains(e.target)) {
```

Comment:
this is my only change outside "flyouts" with an impact. in particular, I'm puzzled how regaining the key handler ever worked, because the click target is as good as never the mainDivRef.